src/charm.py

```diff
@@ -50,6 +51,12 @@ def __init__(self, *args):
             self.framework.observe(self.prometheus_provider.on.targets_changed,
                                    self._on_scrape_targets_changed)
 
+        self.alertmanager_lib = AlertmanagerConsumer(self,
+                                                     relation_name="alertmanager",
+                                                     consumes={'alertmanager': '>0.0.0'})
+        self.framework.observe(self.alertmanager_lib.on.available,
```

Comment:
Very minor style comment. Do you think of the composition here as "a Prometheus object contains annAlertmanager Library" object or "a Prometheus object contains and Altermanager Consumer" object or even just and "Alertmanager" object.

Reply:
The `_lib` follows the graylog pattern. I am trying to think of it as a library for prom.

Reply:
Your call. As I said this is only a style question. I was trying to get you to reflect upon if objects in your programming model compose other objects or libraries ?